micro_manager/micro_manager.py

```diff
@@ -284,7 +284,9 @@ def solve(self) -> None:
                         is_sim_active = np.copy(is_sim_active_cp)
                         sim_is_associated_to = np.copy(sim_is_associated_to_cp)
 
-            else:  # Time window has converged, now micro output can be generated
+            if (
+                self._participant.is_time_window_complete()
+            ):  # Time window has converged, now micro output can be generated
```

Comment:
```suggestion
            # Time window has converged, now micro output can be generated
            if self._participant.is_time_window_complete():
```
Formatting looks a bit odd.

Reply:
I leave the formatting to [black](https://github.com/psf/black).